lib/solidus_social/social_configuration.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```

Comment:
I think that we need to remove `app/models/spree/social_configuration.rb` once we move it to `lib`.

Reply:
Removed this file here [6d93219](https://github.com/solidusio-contrib/solidus_social/commit/6d9321941a614f0dd7541e5e00060a02c01dc75b)